code/modules/projectiles/guns/projectile/shotgun.dm

```diff
@@ -80,16 +80,85 @@
 	desc = "A sturdy shotgun with a longer magazine and a fixed tactical stock designed for non-lethal riot control."
 	icon_state = "riotshotgun"
 	mag_type = /obj/item/ammo_box/magazine/internal/shot/riot
-	sawn_desc = "Come with me if you want to live."
+	var/style = 1

```

Comment:
You should be using the original `sawn_state` var instead of this.

Reply:
With the changes you requested, I think I will need to add in a long_desc, to make sure the pre-sawn version works, but this is more then do-able